portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,11 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function getData(){
+	const name = document.getElementById('text-input').value; 
+	fetch('/data2?name='+name).then(response => response.text()).then((data) => {
+	document.getElementById('data-display').innerText = data;
```

Comment:
Code within anonymous functions should be indented.

Reply:
Alrighty, so I pushed with it fixed.